ngi_reports/ngi_reports.py

```diff
@@ -60,11 +60,11 @@ def make_reports (report_type, working_dir=os.getcwd(), config_file=None, **kwar
     # Get parsed markdown and print to file(s)
     LOG.debug('Converting markdown to HTML...')
     output_mds = report.parse_template(template)
-    for output_bn, output_md in output_mds.iteritems():
+    for output_bn, output_md in list(output_mds.items()):
         try:
             # Ignore special character in the sring, panda might fail because of that
-            output_md = output_md.encode('ascii', 'ignore')
-            with open('{}.md'.format(output_bn), 'w') as fh:
+            #output_md = output_md.encode('ascii', 'ignore')
```

Comment:
Just to double check, do you want to keep this commented out?

Reply:
Done!